### PR TITLE
output/content/eforms: Add primer

### PR DESCRIPTION
**Pull request by duncandewhurst**
Adds the primer documentation drafted in https://github.com/open-contracting/european-union-support/issues/152#issuecomment-1701332797, with minor changes to the title and introductory sentence.

### CODE REVIEW

File: output/content/eforms/primer.md [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+![Ultimate beneficial owner diagram](img/ultimate_beneficial_owners.svg)
+
+## What's not included
+
+[Notice types](https://github.com/OP-TED/eForms-SDK/blob/develop/notice-types/notice-types.json) X01 and X02 relate to events outside the lifecycle of a contracting process.
+
+* X01 - Notice containing information relevant to Formation or Completion of the liquidation of a [EEIG](https://en.wikipedia.org/wiki/European_economic_interest_grouping)
+* X02 - Notice containing information about Registration, Deletion, Transfer-registration or Transfer-deletion of a European Company or European Cooperative Society
+
+Furthermore, thee following notice types are listed in the regulation's annex but are not described by the eForms SDK.
```
Comment (jpmckinney) [inline output/content/eforms/primer.md:121]: Pre-existing typo

```suggestion
Furthermore, the following notice types are listed in the regulation's annex but are not described by the eForms SDK.
```

File: output/content/eforms/how.md [hunk 2]
```diff
@@ -1,15 +1,5 @@
 # How to use this profile
 
-```{attention}
```
Comment (jpmckinney) [inline output/content/eforms/how.md:3]: Maybe add an admonition here prompting the reader to read the primer first.

**Review by jpmckinney — COMMENTED** [review]
Looks good, just a couple comments.

File: output/content/eforms/how.md (hunk 2 above)
Comment (duncandewhurst) [reply output/content/eforms/how.md:3]: Done in https://github.com/open-contracting/european-union-support/pull/191/commits/3b921d582215e8cac28ee677642ebcd2e98df609

**Review by jpmckinney — APPROVED** [review]

**Pull request merged**